### DISCUSSION TITLE
Students unable to reaccept assignments after deleting repository

### DISCUSSION THREAD

**Discussion by jeroenkeppens** [community/community#69057] [GitHub Education]
### Hi everyone,

One of my students needed to restart their assignment and deleted the repository.  The invitation shows a screen with an option to reaccept the assignment, but clicking it leads to a 500 Internal Server Error.  I've tried to replicate the problem, and encounter the same error.

**Comment by Abkeyev99** [comment]
Click "Marked as answer" Thanks so good :)
